### DISCUSSION TITLE
Breadcrumb outputs JavaScript?

### DISCUSSION THREAD

**Discussion by Jarzka** [felix-berlin/astro-breadcrumbs#258] [Q&A] [answered]
Hi. I included Breadcrumb on my page an noticed that it outpus some JavaScript to the generated HTML. I wonder why this happens since I expected the Breadcrum to generate just simple HTML (and CSS) without any JavaScript.

**Comment by felix-berlin** [comment, marked as answer]
Hi @Jarzka, you're right, astro-breadcrumbs ships a [small JS file](https://github.com/felix-berlin/astro-breadcrumbs/blob/main/src/lib/truncated.ts). This is used to dynamically truncate the crumbs on smaller screens.

In the next Major Version, this Script will be only loaded on optin or optout.
